pkg/services/sqlstore/sqlutil/sqlutil.go

```diff
@@ -30,7 +30,7 @@ func MySQLTestDB() TestDB {
 	}
 	return TestDB{
 		DriverName: "mysql",
-		ConnStr:    fmt.Sprintf("grafana:password@tcp(%s:%s)/grafana_tests?collation=utf8mb4_unicode_ci", host, port),
+		ConnStr:    fmt.Sprintf("grafana:password@tcp(%s:%s)/grafana_tests?collation=utf8mb4_unicode_ci&sql_mode='ANSI_QUOTES'", host, port),
```

Comment:
this unfortunately does not have any effect because even though we call it [here](https://github.com/grafana/grafana/blob/be9af35552651d17e29ceef5c54d4e7a6977dce2/pkg/services/sqlstore/sqlstore.go#L649) when we create the engine we [read again](https://github.com/grafana/grafana/blob/be9af35552651d17e29ceef5c54d4e7a6977dce2/pkg/services/sqlstore/sqlstore.go#L671) the `connection_string` from the configuration

Reply:
my mistake; first we create a new section key with the desirable value and then we load it
please ignore it